packages/twenty-front/tsconfig.build.json

```diff
@@ -18,5 +18,5 @@
     "src/**/*.d.ts",
     "src/**/*.ts",
     "src/**/*.tsx"
-  ]
+, "../twenty-server/src/utils/is-uuid.ts", "../twenty-server/src/utils/is-uuid.utils.spec.ts"  ]
```

Comment:
Commited this although I suppose it should not be this way. I suppose something is wrong with my test file

Reply:
Yes this line seems strange, why would you add this? Sorry I don't understand the goal